src/components/Login.jsx

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import PropTypes, { string } from "prop-types";
+import { NavLink } from "react-router-dom";
+
+import "./login.css";
+
+const Login = (props) => {
+
+    return (
+        <>
+            <h1>Login</h1>
+            <section id="login">
```

Comment:
We generally avoid applying IDs to elements unless necessary. Also, rather style your markup using class and element selectors than styling by ID

Reply:
I think id should be used here because class implies multiple elements using it. There is only one login section in this component. I think it's perfectly valid to select via ID in a stylesheet. I actually think we need to move more away from classes.

Reply:
I would even suggest pure HTML selection, like `:first-of-child` etc. But I think we need the `login` id to prevent leaking styles right?